index/score.go

```diff
@@ -348,7 +356,9 @@ func (d *indexData) scoreFile(fileMatch *zoekt.FileMatch, doc uint32, mt matchTr
 // Unlike standard file scoring, this scoring strategy ignores the individual LineMatch and ChunkMatch scores, instead
 // calculating a score over all matches in the file.
 func (d *indexData) scoreFilesUsingBM25(fileMatch *zoekt.FileMatch, doc uint32, cands []*candidateMatch, cp *contentProvider, opts *zoekt.SearchOptions) {
-	tf := cp.calculateTermFrequency(cands)
+	// TODO(jtibs): fix this, maybe we shouldn't reload the whole file here
```

Comment:
I agree, that is probably too expensive. Should we consider storing the file type in the index?